com.geekbench.Geekbench6.appdata.xml

```diff
@@ -20,13 +21,18 @@
     <screenshots>
         <screenshot type="default">
             <caption>Running the vulkan benchmark</caption>
-            <image type="source">https://raw.githubusercontent.com/razzeee/com.geekbench.Geekbench6/f378d2e1432c5fece2ec9ef73b631c31f0328e8f/screenshot1.png</image>
+            <image type="source">https://raw.githubusercontent.com/X-m7/com.geekbench.Geekbench6/b38b8a195a94abc821c2e18a0decb49cd25329fa/screenshot1.png</image>
```

Comment:
Screenshot should be made with default window controls and with shadows on the window

Reply:
Oh, so the shadows has nothing to do with the padding errors from earlier then, no complaints from me as long as the build works lol.